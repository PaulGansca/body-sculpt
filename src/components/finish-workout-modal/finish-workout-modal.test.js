import { shallow } from 'enzyme';
import React from 'react';
import { Provider } from 'react-redux';
import FinishWorkoutModall from './finish-workout-modal';

let wrapper;

beforeEach(() => {
    const mockProps = {
        getState: jest.fn(),
        subscribe: jest.fn(),
        dispatch: jest.fn(),
    }
    wrapper = shallow(
                <Provider store={mockProps}>
                    <FinishWorkoutModall />
                </Provider>
                );
})

describe('<FinishWorkoutModall />', () => {
  it("expect to render finish workout modal", () => {
    expect(wrapper).toMatchSnapshot();
  })
});