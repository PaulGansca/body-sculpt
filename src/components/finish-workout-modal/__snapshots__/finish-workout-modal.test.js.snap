// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FinishWorkoutModall /> expect to render finish workout modal 1`] = `ShallowWrapper {}`;
